src/account/default.ts

```diff
@@ -124,6 +158,49 @@ export class Account extends Provider implements AccountInterface {
     );
   }
 
+  public async declare(
+    { classHash, contract }: DeclareContractPayload,
+    transactionsDetail: InvocationsDetails = {}
+  ): Promise<DeclareContractResponse> {
+    const nonce = toBN(transactionsDetail.nonce ?? (await this.getNonce()));
+    let maxFee: BigNumberish = '0';
+
+    if (transactionsDetail.maxFee || transactionsDetail.maxFee === 0) {
```

Comment:
maybe here 
```suggestion
    if (transactionsDetail.maxFee && transactionsDetail.maxFee === 0) {
```
?

Reply:
I copied it from `execute`. However, now that I think of it, it should be something like this.

```suggestion
    if (transactionsDetail.maxFee) {
```

I don't see a point of `=== 0`, in this case.